ssi-jwt/src/lib.rs

```diff
@@ -46,6 +52,7 @@ pub fn decode_unverified<Claims: DeserializeOwned>(jwt: &str) -> Result<Claims,
 ///     +-2^53 / (1000000 * 60 * 60 * 24 * 365.25) ~= +-285,
 /// which is centered around the Unix epoch start date Jan 1, 1970, 00:00:00 UTC, giving
 /// the years 1685 to 2255.
+// TODO https://docs.rs/hifitime/latest/hifitime/
```

Comment:
This TODO is just saying to make use of the linked library in the future, rather than our own impl?

Reply:
It was to look into it but that crate is irrelevant, I'll remove it